[2.9 ACRA] - setAcraReportingMode crash - `Context.getResources()` was null

This one is worrying, we could be failing to persist the "do not consent to crash logging"

https://couchdb.ankidroid.org/acralyzer/_design/acralyzer/index.html#/report-details/8b41c411-30fa-4cbd-9167-bdda4b18c0c7

```
java.lang.RuntimeException: Unable to instantiate receiver com.ichi2.anki.services.ReminderService: java.lang.RuntimeException: Unable to create application com.ichi2.anki.AnkiDroidApp: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String android.content.res.Resources.getString(int)' on a null object reference
at android.app.ActivityThread.handleReceiver(ActivityThread.java:3376)
at android.app.ActivityThread.-wrap18(Unknown Source:0)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1780)
at android.os.Handler.dispatchMessage(Handler.java:105)
at android.os.Looper.loop(Looper.java:164)
at android.app.ActivityThread.main(ActivityThread.java:6944)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:327)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1374)
Caused by: java.lang.RuntimeException: Unable to create application com.ichi2.anki.AnkiDroidApp: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String android.content.res.Resources.getString(int)' on a null object reference
at android.app.LoadedApk.makeApplication(LoadedApk.java:993)
at android.app.ActivityThread.handleReceiver(ActivityThread.java:3362)
... 8 more
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String android.content.res.Resources.getString(int)' on a null object reference
at android.content.Context.getString(Context.java:556)
at org.acra.config.DialogConfigurationBuilderImpl.<init>(DialogConfigurationBuilderImpl.java:8)
at org.acra.config.DialogConfigurationBuilderFactory.create(DialogConfigurationBuilderFactory.java:1)
at org.acra.config.BaseCoreConfigurationBuilder.configurationBuilders(BaseCoreConfigurationBuilder.java:5)
at org.acra.config.BaseCoreConfigurationBuilder.getPluginConfigurationBuilder(BaseCoreConfigurationBuilder.java:1)
at org.acra.config.CoreConfigurationBuilder.getPluginConfigurationBuilder(CoreConfigurationBuilder.java:1)
at com.ichi2.anki.AnkiDroidApp.setAcraReportingMode(AnkiDroidApp.java:7)
at com.ichi2.anki.AnkiDroidApp.setProductionACRAConfig(AnkiDroidApp.java:1)
at com.ichi2.anki.AnkiDroidApp.onCreate(AnkiDroidApp.java:5)
at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1125)
at android.app.LoadedApk.makeApplication(LoadedApk.java:989)
... 9 more
at androidx.appcompat.view.WindowCallbackWrapper.onCreatePanelMenu(WindowCallbackWrapper.java:1)
```

https://github.com/ACRA/acra/blob/master/acra-core/src/main/java/org/acra/config/BaseCoreConfigurationBuilder.java

The crashing class doesn't seem to exist in the latest version of the ACRA source.

--- a/AnkiDroid/src/main/java/com/ichi2/anki/AnkiDroidApp.java
+++ b/AnkiDroid/src/main/java/com/ichi2/anki/AnkiDroidApp.java
@@ -195,6 +195,14 @@ public class AnkiDroidApp extends Application {
     @Override
     public void onCreate() {
         super.onCreate();
+        if (sInstance != null) {
+            Timber.i("onCreate() called multiple times");
+            //5887 - fix crash.
+            if (sInstance.getResources() == null) {
+                Timber.w("Skipping re-initialisation - no resources. Maybe uninstalling app?");
+                return;
+            }
+        }
         sInstance = this;
         // Get preferences
         SharedPreferences preferences = getSharedPrefs(this);

